config/smart-proxy-answers.yaml

```diff
@@ -0,0 +1,42 @@
+# Format:
+# <classname>: false - don't include this class
+# <classname>: true - include and use the defaults
+# <classname>:
+#   <param>: <value> - include and override the default(s)
+#
+# Every support plugin/compute class is listed, so that it
+# shows up in the interactive menu
+#
+# See params.pp in each class for what options are available
+---
+foreman: false
```

Comment:
@ehelms, That appears correct.  I was able to remove all of the foreman specific params, since they all default to false.  The updated list now only params for foreman_proxy.